sdk/nodejs/docker.ts

```diff
@@ -453,11 +450,8 @@ async function dockerBuild(
     if (build.target) {
         buildArgs.push(...["--target", build.target]);
     }
-    if (build.cacheFrom) {
-        const cacheFromImages = await cacheFrom;
-        if (cacheFromImages && cacheFromImages.length) {
-            buildArgs.push(...["--cache-from", cacheFromImages.join()]);
-        }
+    if (cacheFrom.length) {
+        buildArgs.push(...["--cache-from", cacheFrom.join()]);
     }
```

Comment:
the check of `build.cacheFrom` is unnecessary.  We would only have a non-empty `cacheFrom` value if `build.cacheFrom` was there.